easybuild/easyconfigs/t/TRIQS/TRIQS-3.1.0-gofb-2021a.eb

```diff
@@ -62,11 +53,8 @@ sanity_check_commands = [
     "python -c 'import triqs'"
 ]
 
-modextrapaths = {
-    'CPLUS_INCLUDE_PATH': 'include',
-    'PYTHONPATH': 'lib/python%(pyshortver)s/site-packages',
-    'CMAKE_PREFIX_PATH': ['lib/cmake/triqs', 'lib/cmake/cpp2py']
-}
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
```

Comment:
We usually don't fiddle with PYTHONPATH, except in very rare and difficult cases. 
Instead: `modextrapaths = {"EBPYTHONPREFIXES": ""}`.